superset-frontend/cypress-base/cypress/integration/dashboard/utils.ts

```diff
@@ -85,6 +85,7 @@ export const nativeFilterTooltips = {
   multipleSelect: 'Allow selecting multiple values',
   defaultValue:
     'Default value must be set when "Filter value is required" is checked',
+  preFilter: `Limits the number of selectable values in a filter. This is useful when you want to display a subset of the column's values either for performance reasons or business rules. Some examples include limiting the number of people displayed to a particular business unit or showing only people registered in the last 90 days.`,
```

Comment:
@michael-s-molina if it's ok with you I might tweak the copy of the this tooltip, i.e., make it more concise.

Reply:
No problem @john-bodley. Just suggest an improved copy.

Reply:
I agree with @john-bodley , I think this could be less verbose. Could something like this work?

> Add filters clauses to control the filter's source query. This is useful when you want to improve the query's performance by only scanning a subset of the underlying data or limit the available values displayed in the filter.

Reply:
@michael-s-molina and @villebro I _think_ this is more accurate:

> Add filter clauses to control the filter's source query, though only in the context of the autocomplete i.e., these conditions do not impact how the filter is applied to the dashboard. This is useful when you want to improve the query's performance by only scanning a subset of the underlying data or limit the available values displayed in the filter.